### PR TITLE
DEI translations-   Event diversity 

### PR DESCRIPTION
**Pull request by eyehwan**
https://github.com/chaoss/translations/issues/36

### CODE REVIEW

File: chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## 描述
+
+与会者和讲演者的人口统计学特征有助于表明活动中不同观点和更广泛的视角和代表性的潜力。因为每个人都有不同的观点，调查每个人口统计群体的反应可以表明，来自不同人口统计群体的人们是否觉得自己的被包容度低于平均值。
+
+## 目标
+
+- 确定与会者、演讲者和志愿者的[多元化](https://github.com/drnikki/open-demographics)，包括性别、年龄、母语和有无残疾。
+- 帮助留住来自不同背景的与会者、演讲者和志愿者，为未来的活动做准备。
+- 根据主题、会议和路线分组时，确定演讲者组是否多样化。
```
Comment (xiaoya-Esther) [inline chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md:13]: propose：根据主题、会议和路线分组时，确定**一组**演讲者是否多样化

File: chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### 数据收集策略
+
+- [demographics]during conference registration请求与会者和演讲者在会议登记时做[人口统计调查](http://nikkistevens.com/open-demographics/) 
+- 在活动之前、期间或之后使用调查来收集与会者和演讲者的人口统计资料 (例如，使用 [Open Demographics questions](http://nikkistevens.com/open-demographics/))
+- 对于线上活动: 如果线上活动平台支持实时问卷调查，那么使用问卷调查可以更好地了解参加活动的人的背景
+ 
+## 参考资料
+
+*感谢 Nikki Stevens 和 Marie Nordin 分享经验和资源，这些经验和资源使得这个度量标准得以创建。*
```
Comment (xiaoya-Esther) [inline chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md:41]: 度量标准 -> 指标

File: chinese/wg-diversity-inclusion/focus-areas/event-diversity/time-inclusion.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## 描述
+
+全局可访问性在选择活动发生的时间方面具有重要的作用。线上活动的时区包容性确保了活动期间全球社区的参与。通过这一指标，我们能够更加包容世界各地的与会者和演讲者。 
+
+## 目标
+ 在举办线上活动时，应了解全球无障碍环境，以便让所有区域的与会者和发言者都能从活动中受益。这个度量标准的目的是帮助线上活动的组织者制定一个合理的方法，以确保那些在不同时区的人感到被包括在内。
+
+## 实现
+
+可实施的时区包容性的例子包括::
```
[the remark this answers is not in the document]
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/event-diversity/time-inclusion.md:14]: Done

File: chinese/wg-diversity-inclusion/focus-areas/event-diversity/time-inclusion.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- 让发言者做一个事先录音，发言者可以在聊天中实时回答问题
+- 在特定时区的特定时间段内进行演讲
+- 准备并发送一段录音，在一小时内完成直播
+- 平台的网络带宽选择，用于现场参与和日后的视频播放
+
```
[the remark this answers is not in the document]
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/event-diversity/time-inclusion.md:19]: Done

File: chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md (hunk 1 above)
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md:13]: Done

File: chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md (hunk 2 above)
Comment (eyehwan) [reply chinese/wg-diversity-inclusion/focus-areas/event-diversity/event-demographics.md:41]: Done

**Comment by xiaoya-Esther** [comment]
Thanks @eyehwan

**Pull request merged**